content/composer/overview.md

```diff
@@ -7,28 +7,32 @@ weight: 200
 
 ---
 
-# Introduction
+## Introduction & Product Overview 
+
 The Blueprint Composer is a graphical editor for creating blueprint YAML files dynamically using
-a modern Drag and Drop Interface, including network items, custom types and inline types and
-custom plugins and interfaces. The composer enables topology modeling for complex
-applications while providing means to add relevant lifecycle operations implementation via
-external plugins and scripts. The end result will be a downloadable file generated by the
-composer which will provide a TOSCA based description for the application topology and
-lifecycle management.
-
-# Login
-Upon first use of the Blueprint Composer the user will be asked to enter a username. After first
-login, the user will be automatically routed to the Composer UI unless a logout was performed.
-A logged in user will view his stencils (details in relevant section) and will be able to save/load
-his blueprints for further processing in following sessions.
-
-# Topology
-The topology section allows you to add nodes to the blueprint. Simply choose a node type on
-the left and drag it to the canvas on the right. This will add the node to the blueprint. To delete
-it, click on the node to open the properties panel on the right, and click on the Delete button.
+a modern drag and drop interface. <br />
+The composer enables topology modeling for complex applications while providing means to add relevant lifecycle operations implementation via external plugins and scripts.  <br />
+
+Among its draggable components you'll find platform & network items, you'll be able to add custom & inline types, and you'll be able to use your own custom plugins and interfaces. <br />
+
+The generated result will be a downloadable tgz file containing:  <br />
+- blueprint.yaml file which provides a TOSCA based description for the application topology and its lifecycle management. <br />
+- Custom types and plugins (if added be the user) and other resources the blueprint.yaml depends on.
+
+### Login / Logout
+Upon first use of the Blueprint Composer the user will be asked to enter a username.<br />
+After first login, the user will be automatically routed to the Composer UI.<br />
+A logged in user will be able to save/load his blueprints for further processing in following sessions.<br />
+
+Logout will destroy the previous user session.   
+
```

Comment:
I would replace "After first login... user session" with:
"After login, the user enters the composer UI and can start creating new blueprints. Blueprints can be saved and re-loaded again in future sessions of this user, but they will not available to other users."
The sentence about the "destroy" is a bit scary :smile: